### PR TITLE
Firefox 48 compatibility tests

### PR DESCRIPTION
**Pull request by EnTeQuAk**

### CODE REVIEW

File: validator/testcases/regex.py [hunk 1]
```diff
@@ -426,6 +426,42 @@ def js_tests(self):
...
 
+@register_generator
+class Gecko48RegexTests(CompatRegexTestHelper):
+    """Regex tests for Firefox 48 updates."""
+
+    VERSION = FX48_DEFINITION
+
+    def js_tests(self):
+        yield self.get_test(
+            r'\bProxy\.(create|createFunction)\b',
```
Comment (wagnerand) [inline validator/testcases/regex.py]: I am afraid this is bound to create a huge amount of false positives.

**Comment by wagnerand** [comment]
r+

File: validator/testcases/regex.py (hunk 1 above)
Comment (EnTeQuAk) [inline validator/testcases/regex.py]: I fixed this in 3dcebf8 to not use a regex which should produce much less false positives.

**Comment by wagnerand** [comment]
r+ with ❤️

**Comment by EnTeQuAk** [comment]
Awww :smiley:

**Pull request merged**